plugins/bazaar-backend/migrations/20210902073122_init.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+exports.up = async function setUpTables(knex) {
+  await knex.schema.createTable('metadata', table => {
+    table.comment('The table of Bazaar metadata');
+    table.text('name').notNullable().comment('The name of the entity');
+    table.text('entity_ref').notNullable().comment('The ref of the entity');
```

Comment:
Yes, currently one bazaar item is tied to one entity, since we thought that would make the transition between "regular" entity and "bazaar" entity easier. So yes you would have to create an entity from a template first to be able to add it to the bazaar.

Do you think it would be better to not require a bazaar item to be tied to an entity?